microsoft/iot-cardboard-js
# fix: OAT: Handle targets that are arrays

### Summary of changes 🔍 
DTDL does allow for the target field to be an array of length 1 or to be a string. Today we had always assumed it was a string, but now we will be resilient to receiving an array.

Before
![image](https://user-images.githubusercontent.com/57726991/220485375-778b56fd-14ac-48e8-b9ea-98ecadf081ee.png)

After
![image](https://user-images.githubusercontent.com/57726991/220485306-6b1e681d-d4d5-478a-822e-a48cb64bca3e.png)


### Testing 🧪
Sample model from the bug report - [Link](https://github.com/Azure/iot-plugandplay-models/blob/main/dtmi/com/socomec/socomecgateway-1.json)

- update the name of the targeted model, see that the relationship between the nodes is retained
- Import a model that has a target with an array instead of string
